interceptors/logging/examples/kit/example_test.go

```diff
@@ -58,13 +45,59 @@ func ExampleInterceptorLogger() {
 	_, _ = grpc.Dial(
 		"some-target",
 		grpc.WithChainUnaryInterceptor(
-			logging.UnaryClientInterceptor(InterceptorLogger(logger), opts...),
+			logging.UnaryClientInterceptor(examplekit.InterceptorLogger(logger), opts...),
 			// Add any other interceptor you want.
 		),
 		grpc.WithChainStreamInterceptor(
-			logging.StreamClientInterceptor(InterceptorLogger(logger), opts...),
+			logging.StreamClientInterceptor(examplekit.InterceptorLogger(logger), opts...),
 			// Add any other interceptor you want.
 		),
 	)
 	// Output:
 }
+
+type kitExampleTestSuite struct {
```

Comment:
Let's avoid "suite" patterns, especially for single method test. Why not single unit test?

Reply:
I used the test suite because that already has a server and client I can easily use for a test. I try to find another way to do that

Reply:
if you have any idea how to that this without test suite I will try to implement that.

Reply:
> Let's avoid "suite" patterns, especially for single method test. Why not single unit test?

Hello, pardon my curiosity but I wonder why should we avoid suite pattern since it gave us a server and client inside?
It gave us expandable way to add more tests in the future.

Reply:
Test suite types in Go like this are uncommon and unnecessary. If you need a server in a test, you can just create one. If you want to isolate the logic, create a function. Test suites take OOP beyond what is usually necessary in Go. Just write test functions, skip the suite.